app/src/server/scripts/zeroCheckedToZeroInvoice.ts

```diff
@@ -0,0 +1,40 @@
+import { getPreviousMonthPeriodUTC } from "~/utils/dayjs";
+import { prisma } from "../db";
+import { createInvoice } from "../tasks/generateInvoices.task";
+
+const [startOfPreviousMonth, endOfPreviousMonth] = getPreviousMonthPeriodUTC();
+
+const projectId = process.argv[2];
+
+console.log("Doing: apply credits for all of their usage, then regenerate the invoice");
+
+const project = await prisma.project.findFirstOrThrow({
+  where: {
+    id: projectId,
+  },
+});
+
+const invoice = await prisma.invoice.findFirst({
+  where: {
+    projectId: project.id,
+  },
+});
+
+if (invoice) {
+  await prisma.creditAdjustment.create({
```

Comment:
I tried this in production and found that the invoices are still getting being generated for the original amount. I think the issue is that this `creditAdjustment` needs to be set to be during the invoice period, otherwise it won't be applied to the invoice.